SharpXMPP.Xamarin/SharpXMPP.Client/SharpXMPP.Client.csproj

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>netstandard1.6</TargetFramework>
-		<NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard2.0' ">2.0.0</NetStandardImplicitPackageVersion>
+		<TargetFrameworks>netstandard1.6;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Here I have the same question as above. I can't see a point of building for anything other than `netstandard1.6` if we're okay with `netstandard1.6` per se.